maat-court-data-api/src/main/resources/db/changelog/changes/increase_column_various_mla_tables.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE "MLA"."XXMLA_WQ_DEFENDANT" modify FORENAME varchar2(100 char);
```

Comment:
I have updated this script to explicitly set the CHAR limit as they come through as BYTES by default. This doesn't make much difference, it it just for consistency with the current tables.